frontend/src/components/maps/leaflet/FilterBackdrop/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Spinner } from 'react-bootstrap';
+import styled from 'styled-components';
+
+export interface FilterBackdropProps {
+  show?: boolean;
+}
+
+const Backdrop = styled.div`
+  width: 100vw;
+  height: calc(100vh - 117px);
```

Comment:
could use the css variables here in case the header changes height.

Reply:
Also, does this work with a small screen? in case the filter is pushed to multiple lines and becomes taller then normal.

Reply:
There is a slight difference in the height of the cover of the backdrop, and we can solve it by making the backdrop cover the whole viewport @devinleighsmith

Reply:
I'm a little nervous about doing that, does that completely cover all of the navigation controls as well? I guess my concern would be that if the map is taking a while to load the user can't navigate to any other screens.

Reply:
I'm not a fan of locking down the whole app when ever we make a request.  It ends up having the opposite effect on users experience.